docs: update readme en/cn for jobs module features (#5684)
quisquous/cactbot: README.md

To use this module,
point cactbot at **ui/jobs/jobs.html** or use the `Cactbot Jobs` preset.

This module includes 3 part: resource zone at top middle, raidbuff zone at top right, and tracking zone at bottom.

* **resource zone**: HP bars and job-specific resource, along with some special counter for some jobs.
* **tracking zone**: job-specfic important buff/debuff duration, cooldowns and proc.
* **raidbuff zone**: important raidbuffs duration and coming cooldowns.

You can change some of the behavior or appearence via the user panel, e.g. only show the raidbuff zone, or enable compact view.

> However, customization of some behavior like cooldown alert thresholds and element order is not availble for now.

Reviewer: ```suggestion
However, customization of some behavior like cooldown alert thresholds and element order is not available for now.
```

> In this screenshot, RDM jobs UI is shown as example.

Reviewer: ```suggestion
In this screenshot, the RDM jobs UI is shown as an example.
```

> The ingame UI is shown at top and jobs module is shown at bottom.

Reviewer: ```suggestion
The in-game UI is shown at top and the jobs module is shown at bottom.
```
